From 7c2a115e9ffae16e575749ae612583427b240b64 Mon Sep 17 00:00:00 2001
From: Alexander Zavyalov <whmordred@gmail.com>
Date: Mon, 26 Feb 2024 14:15:02 +0800
Subject: [PATCH] Fix: Chuwi UBook X (CWI535) screen rotation matrix

(cherry picked from commit e66c7eb6dbde36cc15fe043b52fe6e1188a0b91a)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index bbdb217440..ce800da5e4 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -311,7 +311,7 @@ sensor:modalias:acpi:BOSC0200*:dmi:*:svnHampoo*:pnC3W6_AP108_4GB:*
 
 # Chuwi Ubook X (CWI535)
 sensor:modalias:acpi:MXC6655*:dmi*:svnCHUWIInnovationAndTechnology*:pnUBookX:*
- ACCEL_MOUNT_MATRIX=0, 0, -1; 1, 0, 0; 0, 1, 0
+ ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, -1
 
 #########################################
 # Connect
